rules/macos/execution_suspicious_jar_child_process.toml

```diff
@@ -0,0 +1,48 @@
+[metadata]
+creation_date = "2021/01/19"
+maturity = "production"
+updated_date = "2021/01/19"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies suspicious child processes of a Java Archive (JAR) file. JARs may be used to deliver malware in order to
+evade detection.
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Suspicious JAR Child Process"
+risk_score = 47
+rule_id = "8acb7614-1d92-4359-bfcf-478b6d9de150"
+severity = "medium"
+tags = ["Elastic", "Host", "macOS", "Threat Detection", "Defense Evasion"]
+type = "eql"
+
+query = '''
+process where event.type in ("start", "process_started") and
+  process.parent.name:"java" and
+  process.name:("sh", "bash", "dash", "ksh", "tcsh", "zsh", "curl", "wget") and
```

Comment:
maybe to reduce bit noise can make it a sequence network followed by child shell both from java process:
```
sequence by host.id, user.name
 [network where process.name:"java"] by process.entity_id
 [process where event.type in ("start", "process_started") and process.parent.name:"java" and
  process.name:("sh", "bash", "dash", "ksh", "tcsh", "zsh", "curl", "wget") and process.args:"-jar" and process.args:"*.jar"] by process.parent.entity_id
```

Reply:
That's a decent idea. I haven't had an issue with noise on this one. I'd need a bit more telemetry to determine if there's always a network connection event, I haven't seen that. It may be a bit too restrictive if we switch to a sequence.